litmus-portal/frontend/src/pages/ConnectTarget/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Paper, Typography } from '@material-ui/core';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import BackButton from '../../components/Button/BackButton';
+import Scaffold from '../../containers/layouts/Scaffold';
+import TargetCopy from '../../views/Targets/TargetCopy';
+import useStyles from './styles';
+
+const ConnectTarget = () => {
```

Comment:
```suggestion
const ConnectTarget: React.FC = () => {
```
The type can be added as a FC